web/app/Services/Wikitext/HtmlMeta.php

```diff
@@ -3,42 +3,24 @@
 
 namespace Wikijump\Services\Wikitext;
 
-use \FFI;
-
 /**
- * Class HtmlMeta, representing a returned 'struct ftml_html_meta' object.
+ * Class HtmlMeta, represents one &lt;meta&gt;, to be included in a rendered HTML document.
```

Comment:
```suggestion
 * Class HtmlMeta, represents one <meta>, to be included in a rendered HTML document.
```

I assume PHP allows angle brackets in comments?

Reply:
No it interprets HTML elements as elements, like bolding a section of your phpdoc. Given that having a random open `<meta>` tag is not valid, we have to escape the angle brackets.